script/applicant_outreach_report.rb

```diff
@@ -0,0 +1,83 @@
+field_names = %w[
+    primary_hbx_id
+    first_name
+    last_name
+    communication_preference
+    primary_email_address
+    home_address
+    mailing_address
+    application_aasm_state
+    application_aasm_state_date
+    external_id
+    user_account
+    last_page_visited
+    program_eligible_for
+    health_plan_hios_id
+    dental_plan_id
+    subscriber_indicator
+    transfer_id
+  ]
+
+file_name = "#{Rails.root}/applicant_outreach_report.csv"
+
+# Are there times of the year, e.g., during open enrollment, that we would not want to use the current year when looking for enrollments?
```

Comment:
is this comment still valid ? shall we clear with Sarah ?

Reply:
Good catch!  Yes, we should double check this with Sarah.